process/due-diligence-guidelines.md

```diff
@@ -0,0 +1,142 @@
+* do as much reading up as you need to (and consult with experts in the specific field) in order to familiarize yourself with the technology
+  landscape in the immediate vicinity of the project (and don't only use the proposal and that project's documentation as a guide in this regard).
+* at this point you should have a very clear technical idea of what exactly the project actually does and does not do, roughly how it compares with and differs from 
+  similar projects in it's technology area, and/or a set of unanswered questions in those regards. 
+* go through the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/) and for each item,
+  decide for yourself whether or not you have enough info to make a strong, informed call on that item.
+  * If so, write it down, with motivation.
+  * If not, jot down what information you feel you're missing.
+  * Also take note of what unanswered questions the community might have posted in the PR review that you consider
+    to be critically important.
+    
+### Some example questions that will ideally need clear answers
+
+Most of these should be covered in the project proposal document.  The
+due diligence exercise involves validating any claims made there,
+verifying adequate coverage of the topics, and possibly summarizing
+the detail where necessary.
+
+#### Technical
+
+* An architectural, design and feature overview should be available.
+  ([example](https://github.com/docker/notary/blob/master/docs/service_architecture.md),
+  [example](https://github.com/docker/notary/blob/master/docs/command_reference.md))
+* What are the primary target cloud-native use cases?  Which of those:
+  * Can be accomplished now.
+  * Can be accomplished with reasonable additional effort (and are ideally already on the project roadmap).
+  * Are beyond the current roadmap.
+* What are the current performance, scalability and resource consumption bounds of the software?  Have these been explicitly tested?
+  Are they appropriate given the intended usage (e.g. agent-per-node or agent-per-container need to be lightweight, etc)? 
+* What exactly are the failure modes?  Are they well understood?  Have they been tested?  Do they form part of continuous integration testing?
+  Are they appropriate given the intended usage (e.g. cluster-wide shared services need to fail gracefully etc)?
+* What trade-offs have been made regarding performance, scalability, complexity, reliability, security etc?  Are these trade-offs explicit or implicit?
+  Why?  Are they appropriate given the intended usage?  Are they user-tunable? 
+* Code quality.  Does it look good, bad or mediocre to you (based on a spot review).  How thorough are the code reviews? Substance over form.
+  Are there explicit coding guidelines for the project?
+* Dependencies.  What external dependencies exist, do they seem justified, and are their licenses compatible with their usage and CNCF policies?
```

Comment:
I would mention licensing just once, and not in the technical diligence section. You mention it below, also.

Reply:
Agreed, done.